### PR TITLE
chore(topbar-controls): nest controls, css updates

### PR DESCRIPTION
**Pull request by WanderingHogan**
### What this PR does 📖
- We were re-using the same windows-controls component in three places. I moved it to kit.
- Moved the pre-release component to it's own component in ui/src (since it is application specific) and loaded it back in main, and loaded it into the titlebar instead of, hierarchically the same level as titlebar, and removed fixed positioning
- I cleaned up the CSS a little; mac override for a css class in titlebar (to accommodate for the mac windows-controls) and removed things we didnt need.
- removed the 'Uplink" title text. It just got in the way and made that section of the title hard to grab/move

mac
Screenshot 2023-04-18 at 11 51 22 AM

windows
image

### Which issue(s) this PR fixes 🔨

- Resolve #

### Additional comments 🎤

### CODE REVIEW

**Comment by phillsatellite** [comment]
Only issue I found was, if you turn on the dev tools they are spaced out a ton (all os) and push over the window-controls (windows/Linux)

Screenshot 2023-04-18 at 3 56 34 PM
Screenshot 2023-04-18 at 4 00 30 PM

**Comment by WanderingHogan** [comment]
@phillsatellite how is this 
image

File: ui/src/layouts/file_preview.rs [hunk 1]
```diff
@@ -135,35 +132,7 @@ pub fn FilePreview(cx: Scope, file: File, _drop_handler: WindowDropHandler) -> E
 
     #[cfg(not(target_os = "macos"))]
```
Comment (lgmarchi) [inline ui/src/layouts/file_preview.rs]: Hogan, always a new window, we need to do this verification if platform is macOS or not, for topbar controls 

And if we let this responsibility inside TopBarControls, if it is macOS, just return an empty div?

File: ui/src/main.rs [hunk 2]
```diff
@@ -327,43 +331,16 @@ fn auth_wrapper(cx: Scope, page: UseState<AuthPages>, pin: UseRef<String>) -> El
 
     #[cfg(not(target_os = "macos"))]
```
Comment (lgmarchi) [inline ui/src/main.rs]: Same thing here, and soon we are going to have video and voice windows as well.

**Review by lgmarchi — APPROVED** [review]
Just let a suggestion. PR is good to me.

File: ui/src/layouts/file_preview.rs (hunk 1 above)
Comment (WanderingHogan) [reply ui/src/layouts/file_preview.rs]: cool idea i will give it a try

Comment (WanderingHogan) [reply ui/src/layouts/file_preview.rs]: change made

**Review by phillsatellite — APPROVED** [review]

**Comment by phillsatellite** [comment]
Saving for meeting, do not merge

**Pull request merged**